src/manifest.json

```diff
@@ -1,4 +1,6 @@
 {
+    "name": "podman",
```

Comment:
This should not be necessary. If it is for some reason, please split out to a new commit with a rationale.

Reply:
`write_lcov` in test/common/lvoc.py needs this.  I forgot the details, something about constructing filenames in dist/, I think.

Reply:
Eww, what? does that mean that we only get code coverage for the "systemd" webpack in cockpit? That's the only one that has an explicit "name:", due to some historical error/opinion change etc. (I'd really love to just rename "systemd" to "system"..)

Reply:
This is rather curious indeed, however coverage on main seems to work.

https://cockpit-logs.us-east-1.linodeobjects.com/pull-17835-20221024-133642-ee252b8c-fedora-36-devel/Coverage/index.html

Maybe we should instead fallback to `package.json`'name?

Reply:
Does coverage work without this hack? This really bothers me. If this was really required, coverage would be broken for almost all cockpit webpacks as well, and I'd rather fix that in cockpit tests/common then.

Reply:
... in particular as I'd really like to drop/discourage this `name:` thing in manifests.

Reply:
Ah, well this really broke coverage, see here https://cockpit-logs.us-east-1.linodeobjects.com/pull-950-20221025-142943-da2a82ca-fedora-36-devel/log.html

Reply:
The coverage machinery needs to find files in `dist/` when giving their URL.  For example, the machinery needs to make this association:

```
http://127.0.0.2:9091/cockpit/[CHECKSUM]/podman/index.js
  ->
/home/mvo/work/cockpit-podman/dist/index.js
/home/mvo/work/cockpit-podman/dist/index.js.map
```

The URL uses `podman/index.js` while webpack produces just `dist/index.js`.  The coverage machinery needs to know that `podman/index.js` can be found in `dist`, but that for example `base1/cockpit.js` should be ignored.

We can change the build system to produce `dist/podman/index.js`.  This would be quite elegant, but a bit of work.  The starterkit would also need to change.

The "podman" string does in fact come from package.json. (It turns into PACKAGE_NAME in the Makefile, and "make install" puts everything into `/usr/share/cockpit/$(PACKAGE_NAME)`. So we can change lcov.py to use package.json instead if dist/manifest.json doesn't contain "name". (We need to keep looking at dist/*/manifest.json for cockpit-project/cockpit itself.) I think this is a bit magic, but so is the whole distmap thing.

Wdyt? Using package.json seems best for starterkit-based projects.

Reply:
Using package.json sounds good to me.

Reply:
Done, see https://github.com/cockpit-project/cockpit/pull/17849